src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,27 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book zapovit = new Book("zapovit", BigDecimal.valueOf(100));
+        Book newZapiovit = bookDao.create(zapovit);
```

Comment:
The variable 'newZapiovit' is never used. If you don't need the result of 'bookDao.create(zapovit)', you can omit assigning it to a variable.